beta-3-setup.md

```diff
@@ -3762,6 +3762,10 @@ Running ansible:
     export ROUTE_53_WILDCARD_ZONE=cloudapps.example.com
     export ROUTE_53_HOST_ZONE=example.com
 
+### Bootstrap the playbook directories
+
+    ansible-playbook clone_and_setup_repo.yml
+
 ### Configuring the Hosts:
 
     ansible-playbook -i inventory/aws/hosts openshift_setup.yml
```

Comment:
@detiber After doing the step above this needs to be changed to `ansible-playbook -i inventory/aws/hosts/hosts openshift_setup.yml `